src/main/java/com/beerjournal/breweriana/collection/persistence/UserCollectionRepository.java

```diff
@@ -36,40 +33,34 @@
         return crudRepository.findOneByOwnerId(ownerId);
     }
 
-    public Page<ItemRef> findAllInUserCollection(ObjectId ownerId, int page, int count, String name, String category) {
```

Comment:
I believe there is no longer purpose to use java stream for query purpose, in my opinion we should reuse query for lacking items. It could have worse performance but I believe that isn't imporant right now.